web/cobrands/centralbedfordshire/flytipping.js

```diff
@@ -52,4 +55,20 @@ $(fixmystreet).on('report_new:category_change', function() {
 
     checkAndToggle();
     witnessed.on('change', function() { checkAndToggle(); });
+}
+
+$(fixmystreet).on('report_new:category_change', function() {
+    if (!fixmystreet.bodies.includes('Central Bedfordshire Council')) {
```

Comment:
`.includes` [isn't supported](https://caniuse.com/array-includes) in IE11, sadly, so think `.indexOf` is the way to go here.

```suggestion
    if (fixmystreet.bodies.indexOf('Central Bedfordshire Council') === -1) {
```